src/app/src/actions/upload.js

```diff
@@ -60,7 +60,10 @@ export function uploadFile(file = null) {
 
         return csrfRequest
             .post(makeFacilityListsURL(), formData)
-            .then(() => dispatch(completeUploadFile()))
+            .then(({ data: { id }}) => {
```

Comment:
Linter's complaining about `}}` after `id` here, suggesting there needs to be a space between like `id } })`